h/routes.py

```diff
@@ -181,6 +181,8 @@ def includeme(config):  # pylint: disable=too-many-statements
     # Client
     config.add_route("sidebar_app", "/app.html")
     config.add_route("notebook_app", "/notebook")
+    # This URL is temporary. Pending discussions to choose on the final one
```

Comment:
Can you add a link to the relevant GitHub issue, assuming there is one.

Reply:
Done